src/process_wrap.cc

```diff
@@ -163,9 +169,11 @@ class ProcessWrap : public HandleWrap {
       int argc = js_argv->Length();
       // Heap allocate to detect errors. +1 is for nullptr.
       options.args = new char*[argc + 1];
+      CHECK_NE(options.args, nullptr);
```

Comment:
@bnoordhuis The `strdup(…)` ones were actually the ones I was more worried about anyway, because their failure would be going to pass silently and result in inadvertently truncated argument lists/environment variables